docs/topics/request-response.rst

```diff
@@ -24,7 +24,7 @@ below in :ref:`topics-request-response-ref-request-subclasses` and
 Request objects
 ===============
 
-.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback])
+.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback, flags = null])
```

Comment:
Hi, please disregard, I tried pushing this change as part of a branch in
Which i added the flags param. This was mistakenly pushed to another branch.

On Sep 3, 2016 02:30, "Thom Dixon" <notifications@github.com> wrote:

> In docs/topics/request-response.rst
> <https://github.com/scrapy/scrapy/pull/2211#discussion_r77425036>:
>
> > @@ -24,7 +24,7 @@ below in :ref:`topics-request-response-ref-request-subclasses` and
> >  Request objects
> >  ===============
> >
> > -.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback])
> > +.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback, flags = null])
>
> I'm confused. Request doesn't have a flags parameter
> <https://github.com/scrapy/scrapy/blob/73a5571c6044d5aea47b4f973e325c2f3d4e25dc/scrapy/http/request/__init__.py#L19>.
> Moreover, I think you meant flags=None, as Python doesn't have null.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scrapy/scrapy/pull/2211/files/3eb09f7483965bc975e82693acc54507b5aa7347#r77425036>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKA76AnoJEDHWXzNUMt8baesZVZvndsEks5qmLGvgaJpZM4JyZgj>
> .
>